examples/bri-3/src/bri/merkleTree/api/dtos/response/merkleTree.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { AutoMap } from '@automapper/classes';
+import MerkleTree from 'merkletreejs';
+
+export class MerkleTreeDto {
+  @AutoMap()
+  id: string;
+
+  @AutoMap()
+  merkleTree: MerkleTree;
```

Comment:
Do we really want to expose merkletreejs object or some data structure we control? Maybe not relevant for the scope of this PR.

Reply:
agree, and also shouldn't id be part of merkleTree too? i think we can just return id as response, should be enough

Reply:
Yes unlikely we will need to return the whole tree, the get route will also likely not be needed. - right now it was being used as a dev tool to view the whole tree upon return in lieu of test forthcoming. can remove now.